### ISSUE TITLE
Lots of warnings with 1.3.0 version

### ISSUE THREAD

**Issue by wavs** [VidCoin/VidCoin-iOS-SDK#1]
Hi,

I've just updated an app with your latest 1.3.0 SDK which works fine.
But, it seems to give a lot of warnings when compiling.
Some of these warnings seem to reference your derived data directory:

```
while processing /Users/ios-dev/Documents/test-app/iOS/test-app/Vendor/VidCoin/VidCoin.framework/VidCoin(VCViewManager.o):
warning: /Users/Corexpert/Library/Developer/Xcode/DerivedData/ModuleCache/1CY1THAZ5STZT/Foundation-A3SOD99KJ0S9.pcm: No such file or directory
while processing /Users/ios-dev/Documents/test-app/iOS/test-app/Vendor/VidCoin/VidCoin.framework/VidCoin(VCViewManager.o):
warning: /Users/Corexpert/Library/Developer/Xcode/DerivedData/ModuleCache/1CY1THAZ5STZT/Foundation-A3SOD99KJ0S9.pcm: No object file for requested architecture
while processing /Users/ios-dev/Documents/test-app/iOS/test-app/Vendor/VidCoin/VidCoin.framework/VidCoin(VCViewManager.o):
warning: /Users/Corexpert/Library/Developer/Xcode/DerivedData/ModuleCache/1CY1THAZ5STZT/UIKit-2LM3EQU7VVY4O.pcm: No such file or directory
while processing /Users/ios-dev/Documents/test-app/iOS/test-app/Vendor/VidCoin/VidCoin.framework/VidCoin(VCViewManager.o):
warning: /Users/Corexpert/Library/Developer/Xcode/DerivedData/ModuleCache/1CY1THAZ5STZT/UIKit-2LM3EQU7VVY4O.pcm: No object file for requested architecture
...
while processing /Users/ios-dev/Documents/test-app/iOS/test-app/Vendor/VidCoin/VidCoin.framework/VidCoin(VCViewManager.o):
warning: Could not resolve external type c:objc(cs)NSObject
while processing /Users/ios-dev/Documents/test-app/iOS/test-app/Vendor/VidCoin/VidCoin.framework/VidCoin(VCViewManager.o):
warning: Could not resolve external type c:objc(cs)NSMutableArray
while processing /Users/ios-dev/Documents/test-app/iOS/test-app/Vendor/VidCoin/VidCoin.framework/VidCoin(VCViewManager.o):
warning: Could not resolve external type c:objc(cs)NSMutableData
while processing /Users/ios-dev/Documents/test-app/iOS/test-app/Vendor/VidCoin/VidCoin.framework/VidCoin(VCViewManager.o):
warning: Could not resolve external type c:objc(cs)NSDateFormatter
while processing /Users/ios-dev/Documents/test-app/iOS/test-app/Vendor/VidCoin/VidCoin.framework/VidCoin(VCViewManager.o):
warning: Could not resolve external type c:objc(cs)NSURLConnection
while processing /Users/ios-dev/Documents/test-app/iOS/test-app/Vendor/VidCoin/VidCoin.framework/VidCoin(VCViewManager.o):
warning: Could not resolve external type c:objc(cs)NSObject
```

It seems to be the same issue at this [one](https://github.com/zendesk/zendesk_sdk_ios/issues/79).
Maybe related to this option "CLANG_ENABLE_MODULES".

**Comment by RobinDCXP** [comment]
Hi @wavs. Thanks for coming back to us regarding this issue.

Our main focus was to publish an early release including support for iOS 9 and Xcode 7, and even though our SDK goes through a thorough testing process, this issue unfortunately made it to production undetected.

Our team is currently looking at it, and we will release a new version fixing it, hopefully early next week. Do note that to ensure our product's quality, even a small fix will have us go through the full validation process with the SDK before we can release it.

Thanks for your patience.

**Comment by RobinDCXP** [comment]
Hi @wavs.

Good news, we fixed the issue and a new release of our SDK is on its way. You may expect it to be available by Thursday, once we complete our validation process.

Thanks again for your feedback.

**Comment by wavs** [comment]
Hi @RobinDCXP
Thanks for your responsiveness.

**Comment by RobinDCXP** [comment]
Issue #1 fixed in release v1.3.1.
